packages/block-library/src/social-links/edit.js

```diff
@@ -22,17 +27,54 @@ const TEMPLATE = [
 	[ 'core/social-link', { service: 'youtube' } ],
 ];
 
-export function SocialLinksEdit() {
+export function SocialLinksEdit( props ) {
+	const {
+		attributes: { className },
+		backgroundColor,
+		setBackgroundColor,
+		iconColor,
+		setIconColor,
+	} = props;
+
+	const logosOnly = className.indexOf( 'is-style-logos-only' ) >= 0;
```

Comment:
When adding the block first time, the className is undefined so this `indexOf` call causes an error. I think you'll need an `className &&` prepended to check it exists.

Reply:
Is optional chaining (?.) ok to use here? It works and it is a little more concise. Happy to change it to use `className &&` if that is preferred.

Reply:
The switch to optional chaining has fixed this issue - @apeatling are you happy to clear this?